ui/js/values.js

```diff
@@ -32,6 +32,8 @@ treeherder.value("thPlatformNameMap", {
     "android-4-4": "Android 4.4",
     "android-4-4-armv7-api11": "Android 4.4 API11+",
     "android-5-0-armv7-api11": "Android 5.0 API11+",
+    "b2gdroid-4-0-armv7-api11": "B2GDroid 4.0 API11+",
+    "android-4-0-armv7-api11-partner1": "Android 4.0 API11+ partner1",
```

Comment:
@lundjordan - to Ed's point:  the longest visible name is 21 chars now, This one would be 27.  Sorry to nitpick on this.  But if you could minimize it a bit, that'd be great.  Can you get it closer to 23?  If not, just do the best you can and we'll deal. :)